kfs-core/src/main/resources/edu/arizona/kfs/workflow/AssetInventoryUnitMaintenanceDocument.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<data xmlns="ns:workflow"
+	xmlns:fo="http://www.w3.org/1999/XSL/Format"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="ns:workflow resource:WorkflowData">
+		<documentTypes xmlns="ns:workflow/DocumentType"
+			xsi:schemaLocation="ns:workflow/DocumentType resource:DocumentType">
+			<documentType>
+				<name>AIUM</name>
+				<label>Asset Inventory Unit</label>
+				<parent>CMSM</parent>
```

Comment:
The KFS 6.0 document workflows typically also include properties active and routingVersion. Please see  kfs/kfs-core/src/main/config/workflow/030_capital_asset/CapitalAssetSimpleMaintenanceDocuments.xml.